docker-hub/api/latest.yaml

```diff
@@ -0,0 +1,528 @@
+                          type: string
+                          enum: [is_active, current_tag]
+                          example: current_tag
+                        tags:
+                          description: Current tags if warning is `current_tag`
+                          type: array
+                          items:
+                            type: string
+                            example: latest
+paths:
+  /v2/users/login:
+    post:
+      tags: [Authentication]
+      summary: Create authentication token
+      operationId: PostUsersLogin
+      description: |
+        Creates an authentication token that can be used to authenticate with the Docker Hub APIs.
+
+        The returned token is used in the HTTP Authentication header like `Authentication: JWT {TOKEN}`
+
+        Most Docker Hub APIs require this token either to consume or to get detailed information (e.g. listing images of the private repository).
+      parameters:
+        - name: body
+          in: body
+          description: Login details
+          required: true
+          schema:
+            $ref: '#/definitions/UsersLoginRequest'
+      responses:
+        200:
+          description: Authentication successful
+          schema:
+            $ref: '#/definitions/PostUsersLoginSuccessResponse'
+        401:
+          description: Authentication failed
+          schema:
+            $ref: '#/definitions/PostUsersLoginErrorResponse'
+  /v2/namespaces/{namespace}/repositories/{repository}/images-summary:
+    get:
+      tags: [Images]
+      summary: Get summary of repository's images
+      description: |
+        Gets the number of images in a repository and the number of images counted as active and inactive
+      operationId: GetNamespacesRepositoriesImagesSummary
+      produces:
+        - application/json
+      parameters:
+        - name: namespace
+          in: path
+          required: true
+          description: Namespace of repository
+          type: string
+        - name: repository
+          in: path
+          required: true
+          description: Name of repository
+          type: string
+        - name: active_from
+          in: query
+          required: false
+          description: | 
+            Sets the time from which an image must have been pushed or pulled to be counted as active
+
+            Defaults to 1 month before the current time
+          type: string
+      responses:
+        200:
+          description: Success
+          schema: 
+            $ref: '#/definitions/GetNamespaceRepositoryImagesSummaryResponse'
+        401:
+          description: Unauthorized - user does not have read access to the namespace
+          schema:
+            $ref: '#/definitions/ErrorResponse'
+  /v2/namespaces/{namespace}/repositories/{repository}/images:
+    get:
+      tags: [Images]
+      summary: Get details of repository's images
+      description: |
+        Limited to users on Pro and Team plans
```

Comment:
Not a blocker - I am hesitant giving too much details on Plan restrictions, because it might change in the future.
Given that we added a sort of blanket statement in the intro, this could be removed here.
Possibly move the blanket statement out of Authentication section (lines 28-29), up to Overview section (between lines 19-20, removing "Additionally" to avoid repetition).
@usha-mandya what do you think?

Reply:
Agree with @FranchommeE on removing pricing details here. We have a link to the UI docs from this page which provides details about the pricing plans. Let's remove the pricing details here please.

Reply:
Done :)